src/components/ec-input-field/ec-input-field.story.js

```diff
@@ -59,6 +59,9 @@ stories
       iconDate: {
         default: text('icon', '', GROUPS.DATE),
       },
+      isInGroup: {
+        default: text('is in group', ''),
```

Comment:
Maybe make this an array `['', 'left', 'right']` so it's clear for anybody what can be passed into this prop.